src/hevm/src/EVM.hs

```diff
@@ -170,7 +170,7 @@ data Cache = Cache
 -- | A way to specify an initial VM state
 data VMOpts = VMOpts
   { vmoptContract :: Contract
-  , vmoptCalldata :: (Buffer, (SWord 32)) -- maximum size of uint32 as per eip 1985
+  , vmoptCalldata :: Buffer -- maximum size of uint32 as per eip 1985
```

Comment:
We assume the length of calldata is limited to a uint32. But this isn't really relied upon here since we don't carry the length around anymore, so I'll remove this comment